packages/jest-reporters/src/__tests__/notify_reporter.test.ts

```diff
@@ -5,21 +5,22 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import type {TestSchedulerContext} from 'types/TestScheduler';
```

Comment:
this was the only `types/` import left in the code base, which is why I converted it to TS